test/extended/util/framework.go

```diff
@@ -1563,7 +1563,7 @@ func RunOneShotCommandPod(
 		}
 
 		if podHasErrored(cmdPod) {
-			return true, fmt.Errorf("the pod errored trying to run the command")
+			return true, fmt.Errorf("the pod errored trying to run the command: %#v", cmdPod.Status)
```

Comment:
/lgtm cancel

use spew to get full dump `Terminated:(*v1.ContainerStateTerminated)(0xc002a40230)`